paket.dependencies

```diff
@@ -1,11 +1,11 @@
 generate_load_scripts: true
 source https://www.nuget.org/api/v2
 
-framework: auto-detect
```

Comment:
Will the build work with `auto-detect`? If this works, please don't change it

Reply:
solved